customers.sql

```diff
@@ -0,0 +1,12 @@
+select 
+   c.id as customer_id 
```

Comment:
use trailing commas

Reply:
for longer queries or sql you want reviewed, it's best to use table aliases that are descriptive so you can follow through the length of the code without scrolling to find what table you're referencing